src/controllers/user.controller.js

```diff
@@ -16,11 +16,15 @@ const SignUp = async (req, res, next) => {
       if (err) {
         return res.status(500).json({ error: err.message });
       }
-
       const body = {
         id: req.user.id,
         username: req.user.username,
         email: req.user.email,
+        gender: '',
+        currency: 'USD',
+        lang: 'EN',
```

Comment:
You can move these default values to the database models and migrations

Reply:
okay

Reply:
done